framework/python/src/test_orc/test_orchestrator.py

```diff
@@ -350,6 +352,14 @@ def _is_module_enabled(self, module, device):
 
     return enabled
 
+  def get_ip_address(self, interface_name):

```

Comment:
Should this method be in the test orchestrator? It seems like it should be in the net_orc or a net utils class?

Reply:
Moved to IPControl class and routed through net_orc instance